libr/debug/debug.c

```diff
@@ -392,7 +392,7 @@ R_API RDebug *r_debug_new(int hard) {
 	r_debug_signal_init (dbg);
 	if (hard) {
 		dbg->bp = r_bp_new ();
-		r_debug_plugin_init (dbg);
+		r_debug_init_debug_plugins (dbg);
```

Comment:
```suggestion
		r_debug_init_plugins (dbg);
```

Reply:
but i would be also happy to go full bersek and do `r_debug_init_debug_plugins_debug` :D